pkgs/applications/graphics/drawio/default.nix

```
@@ -65,6 +71,11 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
+  '' + lib.optionalString stdenv.isDarwin ''
+    mkdir -p $out/{Applications,bin}
+    mv dist/mac*/draw.io.app $out/Applications
+    makeWrapper $out/Applications/draw.io.app/Contents/MacOS/draw.io $out/bin/drawio
+  '' + lib.optionalString stdenv.isLinux ''
```

```suggestion
  '' + lib.optionalString (!stdenv.isDarwin) ''
```

as we'd do the same thing on BSD etc if our Electron packages supported it.